File: StudyEnglishJava/src/main/java/org/example/studyenglishjava/entity/StudyRecord.java
package org.example.studyenglishjava.entity;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.Getter;
import lombok.Setter;

@Setter
@Getter
@Entity
public class StudyRecord {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private Long userId;
    private Long bookId;
    private Long wordId;
    private Integer stage;
    private Integer strange;
    private Long createTime;
    private Long lastReviewTime;
    private Long updateTime;

}

File: LearningEnglish/app/src/main/java/com/example/learningenglish/http/record/StudyRecord.kt
package com.example.learningenglish.http.record
data class StudyRecord (
    val id: Long,
    val userId: Long,
    val bookId: Long,
    val wordId: Long,
    val stage: Int,
    val strange: Int,
    val createTime: Long,
    val lastReviewTime: Long,
    val updateTime: Long
)